app/dao/provider_rates_dao.py

```diff
@@ -9,3 +9,8 @@ def create_provider_rates(provider_identifier, valid_from, rate):
 
     provider_rates = ProviderRates(provider_id=provider.id, valid_from=valid_from, rate=rate)
     db.session.add(provider_rates)
+
+
+@transactional
+def create_sms_rate(rate):
+    db.session.add(rate)
```

Comment:
i don't think we really need this function, especially since it's only used in the test fn create_rate. i'd just stick the db.session.add in there